tactics/tactics.ml

```diff
@@ -1770,6 +1770,13 @@ let descend_in_conjunctions avoid tac (err, info) c =
 (*            Resolution tactics                    *)
 (****************************************************)
 
+let tclORELSEOPT t k =
```

Comment:
If we're no longer returning an option, then I don't think it's makes sense to call this *OPT

Reply:
Right. Actually, I don't know exactly what should be used...

If we don't care about what exceptions are exactly raised, my understanding is that we can directly use `Proofview.tclORELSE`.

Reply:
I think given the absence of data we should do the `assert false` experiment I've proposed above.